Ginger/Ginger/SolutionWindows/AddApplicationPage.xaml.cs

```diff
@@ -103,18 +111,101 @@ private void OKButton_Click(object sender, RoutedEventArgs e)
             {
                 mSolution.ApplicationPlatforms = new ObservableList<ApplicationPlatform>();
             }
+            var name = variableName.Text.Trim();
```

Comment:
Change the name according to application window

Reply:
Resolved this issue